Djeddit/urls.py

```diff
@@ -39,12 +39,10 @@
     path('logout/', views.LogoutView.as_view()),
     path('bio/<slug:user>/', views.bio_view),
     path('p/<int:post>/delete/<int:comment>/', views.delete_comment_view),
-    path('subdelete/<int:subreddit>/', views.DeleteSubView.as_view()),
     path('r/<slug:subreddit>/delete/<int:post>/', views.delete_post_view),
```

Comment:
@Bensch97 make sure you add `path('subdelete/<int:subreddit>/', views.DeleteSubView.as_view()),` back in, commit, and push to this branch.